libsoup v3.6.5, brotli migration, even_odd_versions bot config

h-vetinari opened the pull request:
Built on top of #49

danyeaw committed:
Enable even_odd_versions in bot configuration

Commit:
MNT: Re-rendered with conda-smithy 3.52.1 and conda-forge-pinning 2025.09.02.21.58.2

h-vetinari committed:
recipe clean-ups

h-vetinari committed:
libsoup v3.6.5; use gitlab sources

h-vetinari committed:
add brotli migration

h-vetinari committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.12.08.51.31

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

h-vetinari committed:
tab to space

h-vetinari committed:
switch to new krb5 setup

see https://gitlab.gnome.org/GNOME/libsoup/-/commit/123eb029bf3fa08f3c45678c0a43ae4b930e6631

h-vetinari committed:
add libintl-devel on windows

h-vetinari committed:
cross-compile aarch/ppc

h-vetinari committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.12.10.42.01

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

h-vetinari committed:
fix run_test.sh

h-vetinari committed:
add zlib as test dep

hmaarrfk commented:
@conda-forge-admin please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.10.13.43.2

hmaarrfk committed:
Try to fix windows and alt-arch

hmaarrfk committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.10.13.43.27

Other tools:
- conda-build 25.9.0
- rattler-build 0.47.1
- rattler-build-conda-compat 1.4.6

hmaarrfk commented:
i disabled cross compilation, and it "fixed" linux. but windows is still broken

https://github.com/conda-forge/libsoup-feedstock/pull/56

h-vetinari committed:
MNT: Re-rendered with conda-smithy 3.53.1 and conda-forge-pinning 2025.11.20.23.36.56

Other tools:
- conda-build 25.9.0
- rattler-build 0.50.0
- rattler-build-conda-compat 1.4.9

hmaarrfk commented:
See #61
